src/modules/storage/LedgerStorage.ts

```diff
@@ -218,6 +218,29 @@ export class LedgerStorage extends Storages
         return this.query(sql, [height.toString()]);
     }
 
+    /**
+     * Get the highest block height in this Stoa DB
+     * @returns Returns the Promise. If it is finished successfully the `.then`
+     * of the returned Promise is called with the block height
+     * and if an error occurs the `.catch` is called with an error.
+     */
+    public getBlockHeight (): Promise<Height>
+    {
+        return new Promise<Height>((resolve, reject) =>
+        {
+            let sql = `SELECT MAX(height) as height FROM blocks`;
```

Comment:
How about the query below?
`SELECT value FROM information WHERE key = 'height';`

Reply:
That's good, but I want to check the actual stored data.